vyper/semantics/types/module.py

```diff
@@ -79,10 +80,10 @@ def abi_type(self) -> ABIType:
         return ABI_Address()
 
     def __str__(self):
-        return self._id
+        return Path(self._id).stem
 
     def __repr__(self):
-        return f"interface {self._id}"
+        return f"interface {Path(self._id).stem}"
```

Comment:
i think we want to change the str implementation, but not repr -- repr should attempt to be injective back to interfaces

Reply:
i changed _pp_signature to use `str`, i think that should be okay cause its only used for error messages